client/extensions/wp-job-manager/state/setup/selectors.js

```diff
@@ -28,3 +39,14 @@ export function isCreatingPages( state, siteId ) {
 export function shouldGoToNextStep( state, siteId ) {
 	return get( getSetupState( state ), [ 'nextStep', siteId ], false );
 }
+
+/**
+ * Returns true if we should show the setup wizard, false otherwise.
+ *
+ * @param  {Object}  state Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean} Whether to show the setup wizard
+ */
+export function shouldShowSetupWizard( state, siteId ) {
+	return get( getSetupState( state ), [ 'status', siteId ], false );
```

Comment:
I wonder if maybe "status" is not descriptive enough? To me, it's not perfectly clear that `status: { [ siteId ]: false }` should mean that the setup wizard should no longer be shown. Do you think it would be better to have a more descriptive name for the property?

Alternatively, maybe it makes sense for the `status` to be an object that contains more properties, rather than a single boolean?

```javascript
{
  status: {
    [ siteId ]: {
      showSetupWizard: false,
      ... // Possibly more status properties in the future
    }
  }
}
```